tests/test_converter.py

```diff
@@ -209,3 +212,43 @@ def test_example_4():
     })
 
     assert qubo == expected
+
+
+def test_to_dqm():
+    num_variables = 2
+    variables = sympy.symbols(
+        " ".join([f"x{i}" for i in range(num_variables)])
+    )
+    objective_function = from_sympy(variables[0] + variables[1])
+    print(objective_function)
+
+    constraint_le = Constraint(objective_function, [Polynomial(1)],
+                               Operator.LE, MethodsForInequalities.SLACKS_LOG_2, "s",)
+
+    problem = SimpleProblem(
+        objective_function, constraint_le,
+        MethodsForInequalities.SLACKS_LOG_2)
+
+
+    dqm = Converter.to_dqm(problem)
+
+    assert isinstance(dqm, DiscreteQuadraticModel)
+
+
+def test_to_cqm():
+    num_variables = 2
+    variables = sympy.symbols(
+        " ".join([f"x{i}" for i in range(num_variables)])
+    )
+    objective_function = from_sympy(variables[0] + variables[1])
+
+    constraint_le = Constraint(objective_function, Polynomial(1),
+                               Operator.LE, MethodsForInequalities.SLACKS_LOG_2, "s",)
+
+    problem = SimpleProblem(
+        objective_function, [constraint_le],
+        MethodsForInequalities.SLACKS_LOG_2)
+
+    cqm = Converter.to_cqm(problem)
+
+    assert isinstance(cqm, ConstrainedQuadraticModel)
```

Comment:
Its really good practice to add a new line to the end of the file ;) and you can make vscode to add this automatically
```suggestion
    assert isinstance(cqm, ConstrainedQuadraticModel)
    
```